ports/samd/boards/ADAFRUIT_ITSYBITSY_M0_EXPRESS/pins.csv

```diff
@@ -0,0 +1,40 @@
+# Pin rows contain Pin number and pin name.
+# Pin rows start with PIN_
+# LED rows start with LED_
+# If the pin name is omitted, the pin number is added as name.
+# Rows for empty entries have to start with '-'
+# Empty lines and lines not starting with PIN_ or LED_ are ignored
+
+PIN_PA11,D0
+PIN_PA10,D1
+PIN_PA14,D2
+PIN_PA09,D3
+PIN_PA08,D4
+PIN_PA15,D5
+-
+PIN_PA21,D7
+-
```

Comment:
I don't understand these `-` entries, what do they do?  Isn't it simpler to just leave these lines out altogether?

Reply:
For Pins, numbers can be used, which are the item number in the pins.csv file. Since there is no D6, a dummy definition line is used. That scheme existed before. I have a newer (not committed) variant, where the pin number just reflects the Port number, so PA12 is pin 12, PB3 is pin 35.,.... That allows accessing pins not defined in pins.csv. But I keep that scheme for the end.

Reply:
I see.  Let's leave it then and use your new scheme later on.